pandas/compat/chainmap_impl.py

```diff
@@ -34,10 +34,10 @@ def wrapper(self):
 
 class ChainMap(MutableMapping):
     """ A ChainMap groups multiple dicts (or other mappings) together
-    to create a single, updateable view.
+    to create a single, updatable view.
 
     The underlying mappings are stored in a list.  That list is public and can
-    accessed or updated using the *maps* attribute.  There is no other state.
```

Comment:
Keep the space after the period.

Reply:
I see that it's because of `flake8` issue.  Perhaps replace the "or" with a slash to get it to fit. :smile:

Reply:
Are 2 whitespaces necessary? It had two, reduced to one.

Reply:
For consistency, yes.

Reply:
Can the slash have spaces before and after?

Reply:
Perfectly fine.